documentation/mqtt-overview.md

```diff
@@ -0,0 +1,83 @@
+# AnkerMake MQTT Communications Overview
+
+We're going over the AnkerMake implementation of the [MQTT communications standard](https://mqtt.org/). While we won't be explaining the MQTT standard itself, we will explain certain aspects of how the protocol works for ease of understanding.
```

Comment:
Anker(Make) didn't implement MQTT - they're using bog standard client and server implementations.

However, they did so some fresh jazz to the bytes going over the protocol.